modules/custom/az_demo/migrations/az_demo_accordion_paragraph.yml

```
@@ -0,0 +1,55 @@
+id: az_demo_accordion_paragraph
+label: AZ Quickstart Demo Accordion Paragraphs
+migration_tags:
+  - Quickstart Demo Content
+  - Content
+dependencies:
+  enforced:
+    module:
+      - az_flexible_page
+source:
+  plugin: url
+  data_fetcher_plugin: file
+  data_parser_plugin: json
+  urls:
+    - profiles/custom/az_quickstart/modules/custom/az_demo/data/az_demo_accordion_paragraph.json
+  item_selector: paragraphs
+  ids:
+    id:
+      type: integer
+  fields:
+    -
+      name: id
+      selector: id
+    -
+      name: az_accordion
+      selector: az_accordion
+    -
+      name: bottom_spacing
+      selector: bottom_spacing
+
+destination:
+  plugin: 'entity_reference_revisions:paragraph'
+  default_bundle: az_accordion
+
+process:
+  field_az_accordion:
+    source: az_accordion
+    plugin: sub_process
+    process:
+      delta: delta
+      title:
+        plugin: skip_on_empty
+        method: process
+        source: title
+      body:
+        plugin: skip_on_empty
+        method: process
+        source: body
+      body_format:
+        plugin: default_value
+        default_value: az_standard
+      collapsed:
+        plugin: skip_on_empty
+        method: process
+        source: collapsed
```

```suggestion
        source: collapsed
  behavior_settings:
    plugin: az_paragraphs_behavior_settings
    paragraph_behavior_plugins:
        az_display_settings:
          bottom_spacing: bottom_spacing
```

Because this paragraph has the default behavior plugin enabled we can do this: https://github.com/az-digital/az_quickstart/blob/main/modules/custom/az_paragraphs/az_paragraphs_accordion/config/install/paragraphs.paragraphs_type.az_accordion.yml#L11C1-L13C18

---

Thanks! That's good to know. I am able to see the the margin being applied based on the data source now. Didn't even think to look TBH